packages/emotion-primitives/src/createEmotion.js

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react'
+import reactPrimitives from 'react-primitives'
+import {
+  channel,
+  contextTypes,
+  setTheme,
+  testAlwaysTrue,
+  pickAssign
+} from './utils'
+
+import { getStyles } from './getStyles'
+import { convertToRNStyles } from './convertToRNStyles'
+
+const primitives = ['Text', 'View', 'Image']
+
+const isValidPrimitive = primitive => primitives.indexOf(primitive) > -1
+
+// Validate and return the primitive
+const getPrimitive = primitive => {
+  if (typeof primitive === 'string') {
+    if (isValidPrimitive(primitive)) {
```

Comment:
both branches were checking `typeof primitive === 'string'`, so I've just merged those 2 (for readability, not perf reasons)